src/pages/verify.tsx

```diff
@@ -0,0 +1,77 @@
+import { useContext } from "react";
+import Navbar from "@/components/public/Navbar";
+import Loading from "@/components/public/Loading";
+
+import Link from "next/link";
+import { NextRouter, useRouter } from "next/router";
+
+import { Typography, Stack, Box } from "@mui/material";
+import { PagePaths } from "enum/pages";
+
+import { userContext } from "supabase/user_context";
+import VerifyChip from "@/components/profile/VerifyChip";
+
+export default function Verify() {
+    const router: NextRouter = useRouter();
+    const userStatus = useContext(userContext);
+
+    if (userStatus.isLoading) return <Loading />;
+    if (!userStatus.user) {
+        router.push(PagePaths.login);
+        return;
+    }
```

Comment:
I design verify page would be accessed by anyone who login although he has already verified, because his friends who don't have account may ask him how to verify, or you have something concern?

Reply:
If someone want to show their friends in the future how to verify national ID then why do we hide verify option in navbar though

Reply:
Oh yes, we don't sth like Q&A (or mini-map). Ok,I will guard verify page from verified user